collective/watcherlist/sample/newsletter.txt

```diff
@@ -182,23 +191,20 @@ Note the different charset and encoding::
     To: guido@example...
     >>> 'QnJlYWtpbmcgw69udMOpcm7DpHRpw7NuYWwgbmV3czogRmlyc3QgaTE4biBuZXdz' in mail or 'Breaking =C3=AFnt=C3=A9rn=C3=A4ti=C3=B3nal news: First i18n news' in mail
     True
-    >>> self.portal.MailHost.reset()
+    >>> portal.MailHost.reset()
 
 We do the last edit another time and first unconfigure the MailHost,
 to see if this indeed prevents sending emails (it will print the mail
 for debugging purposes)::
 
-    >>> ori_smtp_host = self.get_smtp_host()
-    >>> self.configure_mail_host(u'')
```

Comment:
I think we do still want these lines, otherwise the test is invalid.
But this file has test errors anyway, so needs another look.

Reply:
Fixed.